src/applications/representative-appoint/tests/pages/veteran/veteranContactPhoneEmail.unit.spec.jsx

```diff
@@ -11,7 +11,7 @@ describe('Veteran Contact Phone Email page', () => {
   const {
     schema,
     uiSchema,
-  } = formConfig.chapters.veteranInfo.pages.veteranContactPhoneEmail;
```

Comment:
This is now testing the new component in the file for the original one. There should test files for `claimantInfo.pages.veteranContactPhoneEmail` and `veteranInfo.pages.veteranContactPhoneEmailClaimant`